sdcm/nemesis.py

```diff
@@ -4800,6 +4804,33 @@ def clean_unbootstrapped_node():
             self.log.info("Decommission added new node")
             self.cluster.decommission(new_node)
 
+    def disrupt_disable_binary_and_gossip(self):
+        def are_gate_closed_messages_raised():
+            return bool(list(gate_closed_log_reader))
+
+        with nodetool_context(node=self.target_node, start_command="disablebinary", end_command="enablebinary"):
+            self.target_node.run_nodetool("statusbinary")
+            self.target_node.run_nodetool("status")
+            # Wait until cql_server_controller - CQL server stopped ?
```

Comment:
is the comment necessary?

Reply:
Nope, thought we'd check that the line appears in the log, but it's probably overkill. Removed.